packages/cli/src/lib/codegen/CodeGenerator.ts

```diff
@@ -87,6 +88,23 @@ export class CodeGenerator {
       ? await overrides.getSchemaBindConfig(this._config.project)
       : {};
 
+    if (language === "app-py") {
```

Comment:
We should not be doing this in such a hacky way. We have "codegen overrides" for this reason.

See an example here: https://github.com/polywrap/cli/blob/e3f3597e6249e5b03363b2789583f580be690516/packages/cli/src/lib/defaults/language-overrides/wasm/golang/index.ts#L20-L31

Reply:
I am doing it since codegenOverrides doesn't support what I want to do. We need to refactor codegen overrides but I think it should be a separate PR

Reply:
Wait! It seems like it's supported now!!!